cmd/rosa/main.go

```diff
@@ -105,3 +112,29 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func versionCheck() {
+	rprtr := reporter.CreateReporterOrExit()
+
+	currVersion, err := gversion.NewVersion(info.Version)
+	if err != nil {
+		_ = rprtr.Errorf("Could not verify the current version of ROSA.")
+		_ = rprtr.Errorf("You might be running on an outdated version. Make sure you are using the current version of ROSA.")
+		return
+	}
+
+	latestVersionFromMirror, err := rosa.RetrieveLatestVersionFromMirror()
+	if err != nil {
+		_ = rprtr.Errorf("There was a problem retrieving the latest version of ROSA.")
+		_ = rprtr.Errorf("You might be running on an outdated version. Make sure you are using the current version of ROSA.")
```

Comment:
Consider making the above 2 cases a warning rather than error.  You don't know there is a problem.  And if for some reason user is stuck in that situation persistently (e.g. mirror outage), they'll be seeing errors on _every_ command.

Reply:
I dont think we should error for all scenarios without understanding what is the type of the error otherwise all rosa commands will start to fail.